File: graph01/bank_end/ruoyi-system/src/main/java/com/ruoyi/system/domain/Aiarea.java
package com.ruoyi.system.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * ClassName: Aiarea
 * Package: com.ruoyi.system.domain
 * Description:
 *
 * @Author 张鹏
 * @Create 2026/4/4 07:51
 * @Version 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Aiarea {
    private Integer id;
    private Integer guideId;
    private String contentAi;
    private String type;
}

File: graph01/graph_fontend/vo/Aiarea.js
/**
 * Aiarea VO - AI区域内容实体
 */
export class Aiarea {
  constructor(data) {
    this.id = data.id || null;
    this.guideId = data.guideId || null;
    this.contentAi = data.contentAi || '';
    this.type = data.type || ''; // NO_SELF 或 IS_SELF
  }
}
